features/step_definitions/salt_steps.rb

```diff
@@ -442,3 +442,11 @@
   output, _code = $server.run("salt '#{$minion_ip}' pillar.get '#{key}'")
   fail unless output.split("\n")[1].strip == value
 end
+
+When(/^I disable IPv6 forwarding on all interfaces of this "sle-minion"$/) do
```

Comment:
do we need here "sle-minion" ?

We can remove because it's a specific steps so we can just use 

```I disable IPv6 forwarding on all interfaces of the sle-minion/```

The "sle-minion" we use this for suggest that is a variable.